src/detect_breakends/junction_detection.cpp

```diff
@@ -355,22 +355,29 @@ void detect_junctions_in_alignment_file(const std::filesystem::path & alignment_
     {
         case 0: // simple_clustering
             {
-                std::vector<Junction> current_cluster_members = {junctions[0]};
-                int i = 1;
-                while (i < junctions.size())
+                if (junctions.size() > 0)
                 {
-                    if (junctions[i] == current_cluster_members.back())
+                    std::vector<Junction> current_cluster_members = {junctions[0]};
+                    int i = 1;
```

Comment:
This should use a `size_t`. Otherwise the `i < junctions.size()` will give a warning
```suggestion
                    size_t i = 1;
```